project/npda/templates/dashboard/pdu_card.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'dashboard/base_card.html' %}
+{% block card_title %}
+{{ pdu_object.pz_code }}
```

Comment:
I would possibly swap the pz_code and the lead organisation round here. Codes in my mind don't really belong in templates ever but NPDA have drilled these PZ Codes into people so we should put it in but possibly deemphasise